### PR TITLE
Cleaned up gas limit display to remain consistent when loading page & no limit set

### PR DESCRIPTION
**Pull request by HippityHoppp**
Added Skeleton to Gas Limit when loading
Gas limit now also displays as "None Set" when no limit is set rather than not showing in heading.

### CODE REVIEW

File: src/pages/MinerDashboard/Header/MinerDetails.tsx [hunk 1]
```diff
@@ -116,11 +117,22 @@ export const MinerDetails: React.FC<{
...
           <div>Gas Limit:&nbsp;</div>
-          <div>{maxFeePrice}{' '}{feeDetails?.unit}</div>
-        </Item>) : (null)
+          {maxFeePrice && feeDetails ?  <div>{maxFeePrice + " " + feeDetails.unit}</div> :
+          <div>None Set&nbsp;&nbsp;
+            {/*}<Tooltip>
+              <TooltipContent
+                message={'No gas limit has been set. The payment transaction will be executed when the payment limit is reached, regardless of current gas fees.'}
+              />
+            </Tooltip>*/}
```
Comment (AlexSSD7) [inline src/pages/MinerDashboard/Header/MinerDetails.tsx:129]: Remove this commented block, please.

File: src/pages/MinerDashboard/Header/MinerDetails.tsx [hunk 2]
```diff
@@ -116,11 +117,22 @@ export const MinerDetails: React.FC<{
...
         </Item>
         {
-        (maxFeePrice !== undefined && maxFeePrice > 0) ? (
+        (maxFeePrice !== undefined) ? (
         <Item>
           <div>Gas Limit:&nbsp;</div>
-          <div>{maxFeePrice}{' '}{feeDetails?.unit}</div>
-        </Item>) : (null)
+          {maxFeePrice && feeDetails ?  <div>{maxFeePrice + " " + feeDetails.unit}</div> :
+          <div>None Set&nbsp;&nbsp;
```
Comment (AlexSSD7) [inline src/pages/MinerDashboard/Header/MinerDetails.tsx:124]: This should be just gray `None`.

File: src/pages/MinerDashboard/Header/MinerDetails.tsx [hunk 3]
```diff
@@ -116,11 +117,22 @@ export const MinerDetails: React.FC<{
...
+            {/*}<Tooltip>
+              <TooltipContent
+                message={'No gas limit has been set. The payment transaction will be executed when the payment limit is reached, regardless of current gas fees.'}
+              />
+            </Tooltip>*/}
+          </div>}
+        </Item>) : (
+        <Item>
+          <div>Gas Limit:&nbsp;</div>
```
Comment (AlexSSD7) [inline src/pages/MinerDashboard/Header/MinerDetails.tsx:133]: First, this should be "Gas Price Limit". Gas Limit is a completely different thing.

Also, this value name should change depending on the coin selected. With Ethereum it is Gas Price, with Bitcoin it is sat/b, etc. This is why the generic `maxFeePrice` name was chosen in the API. cc @march08

**Review by AlexSSD7 — CHANGES_REQUESTED** [review]
Thank you for the contribution! I've requested some changes.

**Review by AlexSSD7 — APPROVED** [review]
Looks ok to me. @march08 please review this one as well.

**Review by march08 — APPROVED** [review]
Good to go, thanks for PR!

**Pull request merged**